### PR TITLE
Cloud Mechanics UAE Meetup Group

### PR DESCRIPTION
**Pull request by 3tallah**

### CODE REVIEW

File: .DS_Store
Comment (noopman) [inline .DS_Store:1]: This file needs to be removed from the PR, please!

File: 2024/.DS_Store
Comment (noopman) [inline 2024/.DS_Store:1]: This file needs to be removed from the PR, please!

**Review by noopman — CHANGES_REQUESTED** [review]
Hello Mahmoud, thanks for the submission of an amazing Global Azure location in UAE! Wow! Please remove the two accidental .DS_Store files from your PR, and I will approve right away!

File: 2024/.DS_Store
Comment (3tallah) [reply 2024/.DS_Store:1]: @noopman file has been removed
thanks

**Review by noopman — APPROVED** [review]
Excellent. Thank you! Approved.

**Pull request merged**